pytest-embedded/tests/test_base.py

```diff
@@ -622,3 +622,42 @@ def test_unclosed_file_handler(test_input, dut):
         '-x',  # fail at the first fail
     )
     result.assert_outcomes(passed=1024)
+
+
+def test_detect_port_with_cache(testdir):
```

Comment:
since we're testing `port` `target` cache, it's better to place this test function under `pytest_embedded_serial_esp`

Reply:
moved it